app/models/ca_bundle_displays.php

```diff
@@ -2003,6 +2005,54 @@ public function getPlacementsInDisplay($options=null) {
 		return $placements_in_display;
 	}
 	# ------------------------------------------------------
+	private function getCustomDisplayComponents(&$va_available_bundles, $t_instance, $vs_table, $vb_show_tooltips, $vs_format) {
+		// Search for Custom Display Components
+		$va_additional_settings = [];
+		$va_file_list = [];
+		$vs_custom_displays_path = __DIR__ . "/CustomDisplayComponents/";
```

Comment:
Can you turn this into a custom hook call?
Something similar to hookRegisterTaskQueuePluginDirectories and then basically just trawl through the the returned list of directories. Then we can put the actual components in a plugin.

Reply:
So a plugin from PROVCAU-85 can implement a method like:

```php
<?
    public function hookRegisterCustomDisplayComponentDirectories($pa_directories = []):array
    {
        $pa_directories[] = __DIR__ . '/CustomDisplayComponents/';
        return $pa_directories;
    }
```